scrapers/HeavyOnHotties.yml

```
@@ -0,0 +1,23 @@
+name: HeavyOnHotties
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - heavyonhotties.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="vid-title clearfix text-center-mobile"]/h2
+      Date:
+        selector: //span[contains(@class, "released title")]/strong/text()
+        postProcess:
+          - parseDate: Jan 2, 2006
+      Details: //div[@class="col-md-9 col-sm-8 clearfix"]/p
+      Performers:
+        Name: //span[@class="feature title"]//a
+      Image: //video[@id="video"]/@poster
```

```suggestion
      Image:
        selector: //video[@id="video"]/@poster
        postProcess:
          - replace:
              - regex: ^//
                with: "https://"
```
The schema is missing from the image URL (at least for the scenes i tested). You'll notice the error if you try to `save` the scene.

---

Have fixed that, Hmmmm I did not have an issue with the scenes when testing and saving so dunno what was happening there. 
All good now though :-)